### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by omniwired**
Update Mars Protocol

Context:
https://twitter.com/mars_protocol/status/1684216754308894721

### CODE REVIEW

**Comment by omniwired** [comment]
cc bump @ethanfrey @uint

**Comment by omniwired** [comment]
@costa2400 what's the problem with this change?

**Comment by costa2400** [comment]
@omniwired accidentally closed but I reopened.

File: README.md [hunk 1]
```diff
@@ -169,8 +169,8 @@ These projects/contracts are developed and maintained by CosmWasm community.
   the leading software to build your own DAO on CosmWasm chains, quickly
   surpassing Aragon in functionality
 - [mars-protocol/v1-core](https://github.com/mars-protocol/v1-core) - Delphi's
-  "Mars Protocol" is the leading lending protocol on Terra and soon launching on
-  Osmosis
+  "Mars Protocol" is the leading lending protocol on Terra, Osmosis and soon launching on
+  Neutron
```
Comment (maurolacy) [inline README.md:173]:
```suggestion
  "Mars Protocol" is the leading lending protocol on Terra, Osmosis, and soon launching on Neutron.
```

**Review by maurolacy — APPROVED** [review]
LGTM.

**Pull request merged**